tests/simulations/ringtest/create_data.py

```diff
@@ -281,6 +286,42 @@ def make_ringtest_edges():
     }
     make_edges(filename="local_edges_C.h5", edges=edges, wanted_attributes=wanted_attributes)
 
+# For neuromodulation test: Create another A->B edge with different afferent_section_pos w.r.t B->B edge
```

Comment:
if we do not pick the neuromodulation circuit this new file is unused right? In other words, this does not alter the behavior of the previous tests right?